packages/enzyme/src/ReactWrapper.js

```diff
@@ -221,6 +227,27 @@ class ReactWrapper {
     return this.single('instance', () => this[NODE].instance);
   }
 
+  /**
+   * If a `wrappingComponent` was passed in `options`, this methods returns a `ReactWrapper` around
+   * the rendered `wrappingComponent`. This `ReactWrapper` can be used to update the
+   * `wrappingComponent`'s props, state, etc.
+   *
+   * @returns ReactWrapper
+   */
+  getWrappingComponent() {
+    if (this[ROOT] !== this || this[RENDERER] === this[WRAPPING_COMPONENT_RENDERER]) {
+      throw new Error('ReactWrapper::getWrappingComponent() can only be called on the root');
+    }
+    if (!this[OPTIONS].wrappingComponent) {
+      throw new Error('ReactWrapper::getWrappingComponent() can only be called on a wrapper that was originally passed a `wrappingComponent` option');
+    }
+    const node = this[WRAPPING_COMPONENT_RENDERER].getNode();
+    const wrapper = this.wrap(node);
+    privateSet(wrapper, ROOT, wrapper);
+    privateSet(wrapper, RENDERER, this[WRAPPING_COMPONENT_RENDERER]);
```

Comment:
This is a bit hacky. Is it acceptable? We _don't_ want this new `ReactWrapper` to render itself, but we _do_ want it to be a root (so we can `.setProps()` on it.)

We could also decide to not support `.setProps()` and make it a non-root. In that case, we could just get by using `.setState()` to change the context.